README.md

```diff
@@ -12,7 +12,40 @@ Once the internal systems are fully redeveloped this service will be either reti
 
 ## Getting started
 
-_TBD_
+To install, use:
+```
+make build
+```
+To run the tests, use:
+```
+make test
+```
+## Usage
+### Using Docker compose
+
+You can run this service using the docker compose file in [sdc-ci-upload-compose](https://github.com/ONSdigital/sdc-ci-upload-compose/tree/update-env-vars-for-rabbit-adapter) and running `docker compose up -d`. This will run all the necessary services that
+it needs to communicate with. All you will need to do then is put a xlxs file inside the `Documents/ftp` directory and file should be put on the FTP.
+
+### Running Standalone
+
+You can run this service on its own but to do so, some changes will have to be made to environment variables such as `SEFT_FTP_PORT` and `SEFT_PUBLISHER_FTP_FOLDER`. This service also needs
```

Comment:
You mention it needs a rabbit config but then don't mention what config needs to be done.  It might be easier to instead give 2 docker run commands (one for the rabbit and another for the ftp) and have the ports in the command be the same as the defaults in this repo.  For example
```
docker run -d -p 5672:5672 rabbitmq:3-management:latest
docker run -d -p 2021:2021 onsdigital/pure-ftpd:latest
```
I don't think the ftp port is right, but if you can get that right, then it'll be super easy for people to use.

Reply:
Spoke about this, I'm gonna instead direct to the sdc-ci-upload-compose repo and run the `docker-compose up -d sdx-seft-publisher` command to bring up the relevant services